src/libs/actions/Delegate.ts

```diff
@@ -642,6 +644,7 @@ function updateDelegateRole({email, role, validateCode, delegatedAccess}: Update
             onyxMethod: Onyx.METHOD.MERGE,
             key: ONYXKEYS.ACCOUNT,
             value: {
+                isLoading: true,
```

Comment:
when it fails, the loading stopped too, no?
```suggestion
                isLoading: false,
```